widgets/src/stories/Billing-widgets.stories.ts

```diff
@@ -41,6 +42,10 @@ const meta = {
         widget = widgets.byPercentage(widgetArgs as ByPercentageWidgetParams)
         return createWidgetPage(widgetType, widget, WIDGET_COLORS, basicVariants)
 
+      case 'byPercentageOfRevenue':
+        widget = widgets.byPercentageOfRevenue(widgetArgs as ByPercentageOfRevenueWidgetParams)
+        return createWidgetPage(widgetType, widget, WIDGET_COLORS, basicVariants)
```

Comment:
since this widget does not have `v1`, don't use `basicVariants` and instead just use a new set of variants with just `v2`